projects/netgrif-components-core/src/lib/panel/abstract-panel.component.ts

```diff
@@ -49,6 +56,13 @@ export abstract class AbstractPanelComponent implements AfterViewInit, OnDestroy
         }
     }
 
+    showPanelHeader(): boolean {
+        if (!!this.hidePanelHeader) {
+            return false;
+        }
+        return !(!!this.config && this.config['panelHeader'] === 'false');
```

Comment:
`!(this.config.?['panelHeader'] === 'false')`

the `'panelHeader'` constant should be extracted into a constant/enum of some kind

Reply:
Corrected